addOns/openapi/src/main/resources/org/zaproxy/zap/extension/openapi/resources/Messages.properties

```diff
@@ -19,7 +19,7 @@ openapi.automation.dialog.summary = URL: {0}, File: {1}
 openapi.automation.dialog.targeturl = Target URL:
 openapi.automation.dialog.title = OpenAPI Job
 openapi.automation.error.file = Job {0} cannot read file: {1}
-openapi.automation.error.misc = Job {0} target: {1} error: {2}
+openapi.automation.error.misc = Job {0} override: {1} error: {2}
```

Comment:
IMO we should keep target but change the code to properly handle the URL vs override (i.e. if no override pass the URL to the message).

Reply:
ok, will look at that

Reply:
btw, this is not the only message with this problem, `openapi.automation.error.url` has too.

Reply:
Maybe target is not the ideal (was we are calling the field "Target URL"), though override might not be either depending the case.

Reply:
well the text could be "target url"

Reply:
That does not make the message more clear when it's still null.

Reply:
We could drop the word and just have `{0}, {1}` (or a colon instead of comma)

Reply:
Showing just null is less clear IMO.

Reply:
Ok, I need a suggestion here.

Reply:
@psiinon got input on this?

Reply:
The code needs to be changed to not pass null, that's the first step to have clearer message.

Reply:
Sorry I thought `Showing just null is less clear IMO.` was referring to dropping the word being "less clear", not that the value may still be null....

Reply:
Given the current changes, if it's still null what do we want it to say?

Reply:
Or simply don't declare the value null to start with?
```
        String targetStr = this.getParameters().getTargetUrl();
        String targetUrl = null;
```

I guess there's no reason they can't be the same value to start.
```
        String targetStr = this.getParameters().getTargetUrl();
        String targetUrl = targetStr;
```

Reply:
The current changes still allow null to be shown, if we don't have the override we should show either the URL or file.

Reply:
Okay, hopefully 55090a6 is more what you were thinking.

Reply:
The `openapi.automation.error.url` should not be falling back to `apiUrl` since it's already showing the `apiUrl` by its own.
The `openapi.automation.error.misc ` should not say target URL, it's not the target URL in all cases, better keep target.

Reply:
They are both error, did you mean `.url`? That one should just not fallback in L195, it should use `targetUrl` always (assuming we are fine with null in that case, we are showing both so should be IMO).

Reply:
opps yeah

Reply:
Do we want another value when null? Something like "not identified" or _______ ?

Reply:
Ok, I let it be null. If we get that far it's probably fine.